examples/polymorphic/expected/UserOrPost/UserOrPost.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spawnia\Sailor\Polymorphic\UserOrPost;
+
+class UserOrPost extends \Spawnia\Sailor\TypedObject
+{
+    /** @var \Spawnia\Sailor\Polymorphic\UserOrPost\Node\User|\Spawnia\Sailor\Polymorphic\UserOrPost\Node\Post|\Spawnia\Sailor\Polymorphic\UserOrPost\Node\Task */
```

Comment:
Not sure if doable but this could use import for better readability

Reply:
Agreed on the readability. `nette/php-generator` does not support adding imports, so I will hold off on investing extra work into a cosmetic change that may or may not improve the library.